doc/paper/espnet-se++/paper.bib

```diff
@@ -0,0 +1,177 @@
+@article{Watanabe:2018,
+  author={Watanabe, S., Hori, T., Karita, S., Hayashi, T., Nishitoba, J., Unno, Y., ... & Ochiai, T.},
```

Comment:
I think we should add all names instead of `Unno, Y., ... & Ochiai, T.`

Reply:
I think I need to fix some issues here. I will fix it tonight!

Reply:
References have been updated!